test/test_nestedtensor.py

```diff
@@ -475,9 +475,13 @@ def test_nested_namespace(self):
         nested_namespace_result = torch.nested.to_padded_tensor(nt, 4)
         self.assertEqual(result, nested_namespace_result)
 
-    def test_to(self):
+    @parametrize("layout", [torch.strided, torch.jagged], name_fn=layout_name)
+    def test_to(self, layout):
         ntensors = 4
-        nt = random_nt(torch.device('cpu'), torch.float32, ntensors, (4, 4))
+        if layout == torch.strided:
+            nt = random_nt(torch.device('cpu'), torch.float32, ntensors, (4, 4), layout=layout)
+        else:
+            nt = random_nt_from_dims((7, None, 10), torch.device('cpu'), torch.float32, layout=layout)
```

Comment:
nit: is it possible to use `random_nt_from_dims()` for both layouts here and below?